scripts/jest.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-setup_service google-chrome-stable 83.0.4103.61-1
-
 source $OKTA_HOME/$REPO/scripts/setup.sh
```

Comment:
Do we intend to include this in our `.travis.yml` config as well?

Reply:
sounds like a reasonable proposition